### PR TITLE
The Box Model, Block and Inline, Alignment Lessons: Move scrims to Additional Resources

### PR DESCRIPTION
**Pull request by aniket-agra**
## Because

This PR moves Scrimba embeds from their current position to the Additional Resources section as link text as part of [this issue](https://github.com/TheOdinProject/curriculum/issues/26057).

## This PR

- Moves one existing Scrim to Additional Resources for [The Box Model](https://www.theodinproject.com/lessons/foundations-the-box-model) lesson.
- Moves one existing Scrim to Additional Resources for [Block and Inline](https://www.theodinproject.com/lessons/foundations-block-and-inline) lesson.
- Moves two existing Scrims to Additional Resources for [Alignment](https://www.theodinproject.com/lessons/foundations-alignment) lesson.

## Issue

Related to #26057 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/html_css/css-foundations/block-and-inline.md [hunk 1]
```diff
@@ -78,3 +74,4 @@ This section contains helpful links to related content. It isn’t required, so
 
 *   [This tutorial](https://learnlayout.com/no-layout.html) is a little dated at this point, but its examples are clear. The first 6 slides cover the material we've seen so far.
 *   Watch ["this" ](https://www.youtube.com/watch?v=nfXRw06FgK8) simple short video on What does the term "Normal Flow" Mean In CSS
+*  Try [this Scrim](https://scrimba.com/scrim/co5024997a7e46c232d9abe55) for a more interactive explanation and example of different options for block and inline display.
```
Comment (thatblindgeye) [inline foundations/html_css/css-foundations/block-and-inline.md:77]:
```suggestion
*  For a more interactive explanation and example, try this [Scrim on block and inline display](https://scrimba.com/scrim/co5024997a7e46c232d9abe55).
```

File: foundations/html_css/css-foundations/the-box-model.md [hunk 2]
```diff
@@ -73,4 +68,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+*  Try [this Scrim](https://scrimba.com/scrim/cof3d488184abe24ec6258ab4) for a more interactive explanation and example of the box model.
```
Comment (thatblindgeye) [inline foundations/html_css/css-foundations/the-box-model.md:71]:
```suggestion
*  For a more interactive explanation and example, try this [Scrim on the box model](https://scrimba.com/scrim/cof3d488184abe24ec6258ab4).
```

File: foundations/html_css/flexbox/flexbox-alignment.md [hunk 3]
```diff
@@ -93,3 +84,5 @@ This section contains helpful links to related content. It isn’t required, so
 * [Aligning Items in a Flex Container](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container) goes into more depth on the topic of axes and `align-items` vs `justify-content`.
 * This [Flexbox Tutorial](https://www.freecodecamp.org/news/css-flexbox-tutorial-with-cheatsheet/) from freecodecamp is another decent resource.
 * [Flexbox Crash Course](https://www.youtube.com/watch?v=3YW65K6LcIA) is a nice resource by Traversy Media.
+* Check out [this Scrim](https://scrimba.com/learn/flexbox/justify-content-flexbox-tutorial-cVWPacR) for an interactive demo of how `justify-content` and its different properties behave.
+* [This next Scrim](https://scrimba.com/learn/flexbox/align-items-flexbox-tutorial-cJqymH9) covers the behavior of `align-items`, how to perfectly center an element on a page using flexbox and much more.
```
Comment (thatblindgeye) [inline foundations/html_css/flexbox/flexbox-alignment.md:88]:
```suggestion
* For more interactive demos, try the [Scrim on the justify-content property](https://scrimba.com/learn/flexbox/justify-content-flexbox-tutorial-cVWPacR) and the [Scrim on the align-items property](https://scrimba.com/learn/flexbox/align-items-flexbox-tutorial-cJqymH9). Note that these Scrims require logging into Scrimba in order to view.
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Thanks for updating these! Had some requests below. When it comes to link text, we need to avoid being vague with it ("This article", "click here", etc), as it makes it difficult for users navigating with assistive tech (like screen readers) to understand the link purpose without any surrounding context.

**Review by thatblindgeye — APPROVED** [review]

**Comment by aniket-agra** [comment]
Thanks for the suggestions! I have updated the repo with the appropriate changes.

**Pull request merged**